index.js

```diff
@@ -0,0 +1,113 @@
+document.addEventListener("DOMContentLoaded", () => {
+
+})
+const createElements = async (pokemon) => {
+    let ul = document.createElement("ul")
+    let left = document.querySelector("#left")
+    let data = document.querySelector(".data");
+
+    if(left){
+        ul.id = "right"
+    } else {
+        ul.id = "left"
+    }
+
+    let name = document.createElement("li")
+    let h4 = document.createElement("h4")
+    h4.innerText = pokemon.name
+    name.appendChild(h4)
+    ul.appendChild(name)
+
+    let image = document.createElement("li")
+    let img = document.createElement("img");
+    img.src = pokemon.sprites.front_default;
+    image.appendChild(img);
+    ul.appendChild(image);
+
+    let hp = document.createElement("li")
+    hp.innerText = `HP: ${pokemon.stats[5].base_stat}`;
+    ul.appendChild(hp);
+
+    let moves = document.createElement("li")
+    moves.innerText = `Moves: `;
+    ul.appendChild(moves);
+
+    let move1 = document.createElement("li")
+    let firstMove = await axios.get(pokemon.moves[0].move.url)
+    firstMove = firstMove.data;
+    move1.innerText = `${firstMove.name} PP:${firstMove.pp}`;
+    ul.appendChild(move1);
+
+    let move2 = document.createElement("li")
+    let secondMove = await axios.get(pokemon.moves[1].move.url);
+    secondMove = secondMove.data;
+    move2.innerText = `${secondMove.name} PP:${secondMove.pp}`;
+    ul.appendChild(move2);
+
+    let move3 = document.createElement("li")
+    let thirdMove = await axios.get(pokemon.moves[2].move.url);
+    thirdMove = thirdMove.data;
+    move3.innerText = `${thirdMove.name} PP:${thirdMove.pp}`
+    ul.appendChild(move3);
+
+    let move4 = document.createElement("li")
+    let fourthMove = await axios.get(pokemon.moves[3].move.url);
+    fourthMove = fourthMove.data;
+    move4.innerText = `${fourthMove.name} PP:${fourthMove.pp}`;
+    ul.appendChild(move4);
```

Comment:
You could make it a little more efficient and use a for loop and loop through the first 4 elements of the moves array and for each element (so for each move) do what you were writing and rewriting 4 times:
`let moveLi = document.createElement("li")
  let move = await axios.get(pokemon.moves[i].move.url)
    move = move.data;
    moveLi.innerText = `${move.name} PP:${move.pp}`;
    ul.appendChild(moveLi);`

Just imagine if you had to display 40 moves, you wouldn't want to hard-code this 40 times. Just throw one formula in a for loop and you're good to go :)